README.md

```diff
@@ -1,4 +1,4 @@
-<p align="center"><img alt="capi" src="https://github.com/kubernetes-sigs/cluster-api/raw/master/docs/book/src/images/introduction.png" width="160x" /><img alt="capi" src="https://cloud.google.com/_static/cloud/images/favicons/onecloud/super_cloud.png" width="192x" /></p>
+<p align="center"><img alt="capi" src="https://raw.githubusercontent.com/kubernetes-sigs/cluster-api/b75062c95d283880964c0a13482bd82b7523814d/docs/book/src/images/introduction.png" width="160x" /><img alt="capi" src="https://cloud.google.com/_static/cloud/images/favicons/onecloud/super_cloud.png" width="192x" /></p>
```

Comment:
`https://raw.githubusercontent.com/kubernetes-sigs/cluster-api/master/docs/book/src/images/introduction.png` would. be better? (avoid SHA)

Reply:
Yes, I'll change that

Reply:
Yes, thank you.
I made that change